src/app/features/genes/components/gene-comparison-tool/gene-comparison-tool.component.ts

```diff
@@ -966,7 +953,7 @@ export class GeneComparisonToolComponent implements OnInit, AVI, OnDestroy {
     }
 
     if (this.significanceThresholdActive) {
-      params['significance'] = [this.significanceThreshold];
```

Comment:
significance value should not be an array, so this will not be converted to an array